src/point_one/fusion_engine/messages/control.h

```diff
@@ -229,21 +284,22 @@ struct alignas(4) ResetRequest : public MessagePayload {
    * - The navigation engine (@ref RESTART_NAVIGATION_ENGINE)
    * - All runtime data (GNSS corrections (@ref RESET_GNSS_CORRECTIONS), etc.)
    * - Position, velocity, orientation (@ref RESET_POSITION_DATA)
+   * - GNSS ephemeris data (@ref RESET_EPHEMERIS)
    * - Fast IMU corrections (@ref RESET_FAST_IMU_CORRECTIONS)
-   * - GNSS measurement engine (@ref RESTART_GNSS_MEASUREMENT_ENGINE)
    *
-   * Not reset:
+   * Not reset/performed:
    * - Training parameters (slowly estimated IMU corrections, temperature
    *   compensation, etc.; @ref RESET_NAVIGATION_ENGINE_DATA)
    * - Calibration data (@ref RESET_CALIBRATION_DATA)
    * - User configuration settings (@ref RESET_CONFIG)
+   * - Reboot GNSS measurement engine (@ref RESTART_GNSS_MEASUREMENT_ENGINE)
```

Comment:
To be clear, you're saying that now going forward the GNSS measurement engine (Teseo) will get the applicable restart (hot, warm, cold), but won't get a full restart? What does `RESTART_GNSS_MEASUREMENT_ENGINE` mean then going forward?